CONTRIBUTING.md

```diff
@@ -55,14 +47,6 @@ $ git switch -c <branch-name>
 ```
 With this branch checked-out, make the desired changes to the package.
 
-Note that Altair code uses the [black](https://black.readthedocs.io/)
-code formatter and [flake8](https://flake8.pycqa.org/en/latest/) linter
-which you can apply to your modifications by running:
-```
-$ black --diff .  # View changes
```

Comment:
My vote would be to change the Makefile to run `black --diff --color .` instead of `black --check .` to include info about what went wrong so that the contributor can easily fix it:

![image](https://user-images.githubusercontent.com/4560057/210353618-fd77f604-cc8d-4b4d-a1b0-c9198cf58922.png)

What do you think?